chandra_aca/aca_image.py

```diff
@@ -918,6 +925,13 @@ def get_aca_images(
     # Get images
     imgs_table = get_aca_images_func(start, stop, **maude_kwargs)
 
+    # Add an IMGSIZE column if not present (maude)
+    # IMGTYPE 4 -> 8, 1 -> 6, 0 -> 4
+    if "IMGSIZE" not in imgs_table.colnames:
+        imgs_table["IMGSIZE"] = np.zeros(len(imgs_table), dtype=np.int32)
```

Comment:
@javierggt I've initialized this new column to have zero values, but will downstream code ever see zeros out of this?  It isn't immediately clear to me if the IMGTYPEs that _aren't_ 4, 1, 0 ever end up in the images from maude get_aca_images.